src/browser/devtools.js

```diff
@@ -0,0 +1,121 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define((require, exports, module) => {
+
+  'use strict';
+
+  const {DOM} = require('react');
+  const Component = require('omniscient');
+  const {Record, List} = require('typed-immutable/index');
+
+  const DevTools = Record({
+    enableHUD: Boolean(false),
+    enableRemoteDevtools: Boolean(false),
+    console: Record({
+      // FIXME: I don't how to update these list from anywhere in our codebase.
+      log: List(String),
```

Comment:
The problem is not that. The problem is that at anytime in the code, and maybe not in a render function, I want to be able to update the state.

Basically, in devtools.js, I'd like to be able to do something like:

```js
console.log = (...args) => {
  realConsole.log(...args);
  devtools.set('console', devtools.console.push(message));
}
```

but I don't have access to an edit function.